web_development_101/git_basics/introduction_to_git.md

```diff
@@ -37,12 +37,12 @@ By the end of this lesson, you should be able to do the following:
 </div>
 
 ### Additional Resources
-This section contains helpful links to other content. It isn't required, so consider it supplemental for if you need to dive deeper into something.
+This section contains helpful links to other content. Use these resources to dive deeper into Git and Github.
 
 * [Git and Github in plain English](https://blog.red-badger.com/blog/2016/11/29/gitgithub-in-plain-english)
 
 ### Knowledge Check
-This section contains questions for you to check your understanding of this lesson. If you're having trouble answering the questions below on your own, clicking the small arrow to the left of the question will reveal the answers.
+In this section you can check your understanding of the lesson. If you're having trouble answering a question, click the small arrow to the left of it to reveal the answer.
```

Comment:
Again, this is a standard phrase used across the site. If we change it here, it needs to be changed on every other file. I think it is fine as it was. The other suggestions are good though. :+1: